kubernetes/device-plugin/server.go

```diff
@@ -47,9 +50,15 @@ type NvshareDevicePlugin struct {
 }
 
 func NewNvshareDevicePlugin() *NvshareDevicePlugin {
+	socketId:= os.Getenv("NVSHARE_SOCK_ID")
+	if len(socketId) == 0 {
+		socketId = "0"
+	}
+
+	serverSockNew:= strings.Split(serverSock, ".sock")[0]+socketId+".sock"
 	return &NvshareDevicePlugin{
 		devs:   getDevices(),
-		socket: serverSock,
+		socket: serverSockNew,
```

Comment:
I see why you did this, Could the name be something more meaningful?  The next person might add another "New" on the end.

I suggest **ServerSockWithId** or **ServerSockId**